packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -16,31 +15,47 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
           display: 'flex',
           alignItems: tokens.alignItems,
           flexDirection: tokens.flexDirection,
-          paddingHorizontal: globalTokens.size40,
+          paddingHorizontal: tokens.rootHorizontalPadding,
+          paddingVertical: tokens.rootVerticalPadding,
           ...borderStyles.from(tokens, theme),
         },
+        android_ripple: { color: tokens.rippleColor },
       }),
-      ['flexDirection', 'alignItems', ...borderStyles.keys],
+      ['flexDirection', 'rootHorizontalPadding', 'rootVerticalPadding', 'rippleColor', 'alignItems', ...borderStyles.keys],
     ),
     button: buildProps(
       (tokens: RadioTokens) => ({
         style: {
           backgroundColor: 'transparent',
-          width: tokens.radioSize,
-          height: tokens.radioSize,
+          width: tokens.radioOuterCircleSize,
+          height: tokens.radioOuterCircleSize,
           alignItems: 'center',
           justifyContent: 'center',
           borderWidth: tokens.radioBorderWidth,
           borderStyle: tokens.radioBorderStyle,
-          borderRadius: tokens.radioSize / 2,
+          borderRadius: tokens.radioOuterCircleSize / 2,
           borderColor: tokens.radioBorder,
+          padding: tokens.radioViewSize - tokens.radioOuterCircleSize,
```

Comment:
radioViewSize isn't defined on other platforms, which probably results in a rendering issue or a runtime crash.

It feels like it would be simpler to have radioOuterCircleSize and radioOuterCirclePadding instead of both radioViewSize and radioOuterCircleSize and deriving the padding that way, but not a strong opinion.

Reply:
Shifting to margin